.project/goreleaser.yml

```diff
@@ -36,6 +37,14 @@ builds:
         goarch: s390x
       - goos: windows
         goarch: ppc64le
+      - goos: illumos
+        goarch: arm
+      - goos: illumos
+        goarch: arm64
+      - goos: illumos
+        goarch: s390x
+      - goos: illumos
+        goarch: ppc64le
```

Comment:
I don't know much about this OS. Are all of these architectures common, or is there 1 that is more common than others?

Reply:
Thank you for looking at the PR right away.

These four groups are the ignored combinations, because currently only `illumos/amd64` is actually used. That's the only combo that we'd want to target.